lib/Config.js

```diff
@@ -292,4 +295,7 @@ class Config extends EventEmitter {
     }
 }
 
-module.exports = new Config();
+module.exports = {
```

Comment:
this has been done to ease to testing

Reply:
yes, but it breaks code coverage (and 10x more changes than what you are trying to change...)

Alternatively, kind of a hack to make it work may be this:

```
class Config {
   constructor() {
       this.Config = Config; // store the `class` as an attribute
       [...]
   }
}

module.exports = new Config();
```

* When imported as before, we get the singleton "config" object.
* When destructured imports are used, we get `Config` field of the "config" singletion object, which is actually the class of the object: and can thus be used to create other config object